google/ggrc-core
# GGRC-2382: Show cycles count in Active Cycles and History tabs

# Issue description
There is no information about cycles count in Active Cycles and History tabs.

# Steps to test the changes
1. Create one time WF
2. Add task for task group
3. Activate workflow
4. Look at the Active Cycles tab: count is not displayed
Actual Result: there is no count in Active Cycles tab 
Expected Result: Count in Active Cycles tab should be displayed
The same is relevant for History tab after moving tasks to final state.

# Solution description
CMS.Controllers.DashboardWidgets subscribes on count update for invalid counter name ('Cycle' should be replaced with 'cycle:history' and 'cycle:active').

# Sanity checklist



# PR Review checklist
